pay-api/migrations/versions/2024_12_18_695a899f8a25_products.py

```diff
@@ -0,0 +1,65 @@
+"""add products table, and insert product codes and their payment methods
+
+Revision ID: 695a899f8a25
+Revises: 4f3a44eeade8
+Create Date: 2024-12-18 11:32:07.538729
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+# Note you may see foreign keys with distribution_codes_history
+# For disbursement_distribution_code_id, service_fee_distribution_code_id
+# Please ignore those lines and don't include in migration.
+
+revision = '695a899f8a25'
+down_revision = '4f3a44eeade8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
```

Comment:
feel free share your thoughts on this table structure.
now it looks like:
![image](https://github.com/user-attachments/assets/fa4793f9-2997-4167-9a96-682ee1c93185)